### DISCUSSION TITLE
Separate layers/pages/frames of an image?

### DISCUSSION THREAD

**Discussion by SethFalco** [dlemstra/magick-wasm#17] [Q&A] [answered]
Hey!

I have images in formats like PSD, ORA, and GIF, and I'd like to get the individual layers/pages/frames from the images.

I can see on Gitter/Matrix you said to use the `MagickImageCollection` class, but it's not clear to me how to utilize that.
The only use to it that I see is the `#seperate` method, which seems to split it by color channel, if I understand correctly?

Could I get any advice on this? ^-^'

**Comment by dlemstra** [comment, marked as answer]
I thought I already added support for reading images with the `MagickImageCollection` but I only added support for using it with the separate method of the `MagickImage` class. Will now work on adding that.

**Reply by dlemstra** [reply]
I just published a new release that includes support for reading collections.

**Reply by SethFalco** [reply]
Thanks a lot for that! \\(^-^)/
I'll update and check it out soon.
